### DISCUSSION TITLE
I have a repeatable issue, but the test case is not small

### DISCUSSION THREAD

**Discussion by markkimsal** [dompdf/dompdf#2767] [General]
There's something with table cell headers that - when repeated on a second page - have different spacing rules than the cells on the first page.  This seems to only happen with :
 * landscape
 * bold fonts
 * table-border: collapse
 * table td { padding: 5px }
 
I'm not sure how long this "eclectic geek" link will stay up.  But there's a lot of CSS.  I keep removing some of it and those are the 4 things that I see affect this issue.  But, it seem so strange that it only happens on table headers that span 2 pages.

I've managed to work around it by specifying `padding: 5px 10px` - which, for my use case - doesn't alter the appearance of the tables.

https://eclecticgeek.com/dompdf/debug.php?identifier=4f41159dcf25e43d46d0af09e1a4b7c8

My guess is that it has something to do with losing precision of dimensions when copying to the new page.

This behavior wasn't there on 1.0.2.

![image](https://user-images.githubusercontent.com/54099/152658333-2c96de9d-27f9-41c4-9ab5-7f23b3d163d1.png)

![image](https://user-images.githubusercontent.com/54099/152658352-dc35a9b8-b904-4510-9d46-81ecb2491112.png)

**Comment by bsweeney** [comment]
It's probably an issue with how Dompdf is splitting frames for line breaks and/or paging. Specifically, it looks like it's removing trailing/leading space that should be left. The next release is much improved in how it handles frame splitting, which you can see if you render using the git_master. Right now we're planning on pushing the new release out next week.

**Reply by markkimsal** [reply]
It does render correctly on git_master.  Which is funny because I'm trying to reduce my "dev-*" dependencies in my composer.json.  I had it pinned to a specific sha from a previous issue with empty() and the character "0".

I'm looking forward to the next release.  Great job!

**Reply by bsweeney** [reply]
New version released.
